golem-cluster.mini.yaml

```diff
@@ -0,0 +1,55 @@
+# Ray on Golem cluster name
+cluster_name: golem-cluster
```

Comment:
This could be problematic, because different cluster names would be used in webserver, that does not currently support multiple clusters. Until then, keeping same name in all yamls will be more desired.